base/complex.jl

```diff
@@ -254,15 +257,27 @@ end
 
 Compute the complex conjugate of a complex number `z`.
 
+When `conj` is applied to signed complex integers, overflow may occur,
+resulting in the return of a negative value of imaginary part. This overflow
+occurs only when `conj` is applied to the minimum representable value of a
+signed integer. That is, when `x == typemin(typeof(x))`,
+`imag(conj(x * im)) == x < 0`, not `-x` as might be expected.
```

Comment:
This warning seems unnecessary.   We don't have similar warnings for `-` or other basic arithmetic operators.